getting-started/kubernetes/customizations/imageset.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Configure Image deployment by digest
+description: Specify the digests the operator will use to deploy images
+---
+
+{% assign operator = site.data.versions.first.tigera-operator %}
+
+### Big picture
+
+Enable deploying images by docker registry digest.
+
+### Value
+
+Some deployments have strict security requirements that require deploying images by immutable digest instead of tags.
+{{site.prodname}} will not update existing tags but using an immutable digest allows specific images to be reviewed
+and verified by security teams.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+* **ImageSet**
+
+### Concepts
+
+#### Docker repository
+
+A docker respository provides access to docker images referenced by tags or digest.
+
+#### Image tag
+
+Versioned docker images are typically referenced by a tag which is appended to an image reference. Example: `<repo>/<image>:<tag>`. Docker image tags are typically not expected be changed or updated, but this is not required or enforced.
+
+#### Image digest
+
+Docker images, when added to a docker repository, have a unique hash created that can be used to pull a specific version of an image that cannot be changed or updated.
+
+### FAQ
+
+#### What do I need to do to prepare to upgrade my cluster?
+
+When upgrading to a new patch, minor, or major release it will be necessary to create a new [ImageSet]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.ImageSet)
```

Comment:
"patch, minor, or major" can just be "upgrading or downgrading to a new release of Calico" (since patch, minor, or major is all of the types of release)

Reply:
Before upgrading to a new release or downgrading, you must create a new [ImageSet]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.ImageSet) that updates image references and names for the new release. This must be done prior to upgrading the cluster so when the new manifests are applied, the appropriate ImageSet is available.